### PR TITLE
Api 2041 remove arcgis cruft

### PR DESCRIPTION
**Pull request by as-va**
https://vajira.max.gov/browse/API-2041

### CODE REVIEW

**Comment by jblefkove-va** [comment]
See also:
```
$ git grep -i 'arc-gis'
facilities-tests/pom.xml:                <argument>-Darc-gis.url=http://localhost:8666</argument>
facilities/src/main/resources/application.properties:arc-gis.url=unset
make-configs.sh:configValue facilities $PROFILE arc-gis.url 'http://localhost:8666'
```

File: facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java [hunk 1]
```diff
@@ -75,44 +72,6 @@ private void addAccessToCareWaitTimes(MockServerClient mock) {
...
-            addQuery(
-                "/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VBA_Facilities/FeatureServer/0/query"
-                    + "?f=json&inSR=4326&outSR=4326&orderByFields=Facility_Number"
-                    + "&outFields=*&resultOffset=0&returnCountOnly=false"
-                    + "&returnDistinctValues=false&returnGeometry=true&where=1=1"))
-        .respond(
-            response()
-                .withStatusCode(arcgisStatusCode)
-                .withHeader(contentTextPlain())
-                .withBody(contentOf("/arcgis-benefits-facilities.json")));
```
Comment (jblefkove-va) [inline facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java:89]: This file can be removed

File: facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java [hunk 2]
```diff
@@ -75,44 +72,6 @@ private void addAccessToCareWaitTimes(MockServerClient mock) {
...
-            addQuery(
-                "/aqgBd3l68G8hEFFE/ArcGIS/rest/services/NCA_Facilities/FeatureServer/0/query"
-                    + "?f=json&inSR=4326&orderByFields=SITE_ID&outFields=*&outSR=4326"
-                    + "&resultOffset=0&returnCountOnly=false&returnDistinctValues=false"
-                    + "&returnGeometry=true&where=1=1"))
-        .respond(
-            response()
-                .withStatusCode(arcgisStatusCode)
-                .withHeader(contentTextPlain())
-                .withBody(contentOf("/arcgis-cemeteries.json")));
```
Comment (jblefkove-va) [inline facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java:103]: This file can also be removed

**Comment by jblefkove-va** [comment]
`./facilities/src/test/resources/arcgis-benefits.json` and `.../arcgis-cemeteries.json` appear to be cruft

File: facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java (hunk 1 above)
Comment (as-va) [reply facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java:89]: Done.

File: facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java (hunk 2 above)
Comment (as-va) [reply facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java:103]: Done.

**Review by taylorrassmannlibertyits — APPROVED** [review]

**Review by jblefkove-va — APPROVED** [review]

**Review by jnguy-va — APPROVED** [review]

**Pull request merged**